saagieapi/projects/projects.py

```diff
@@ -72,7 +72,7 @@ def get_id(self, project_name: str) -> Dict:
         if project:
             project_id = project[0]["id"]
             return project_id
-        raise NameError(f"Project {project_name} does not exist or you don't have permission to see it.")
+        raise NameError(f"❌ Project {project_name} does not exist or you don't have permission to see it.")
 
     def get_info(self, project_id: str) -> Dict:
```

Comment:
`pprint_result` as parameter ?

Reply:
not needed as this function is not called by other functions